src/components/Map/Map.js

```diff
@@ -1,151 +1,56 @@
-import React, { Component } from 'react';
-
+import React from 'react';
 import { withScriptjs, withGoogleMap, GoogleMap } from 'react-google-maps';
 import { MarkerClusterer } from 'react-google-maps/lib/components/addons/MarkerClusterer';
 import OrganizationMarker from './OrganizationMarker';
 
-const googleMapKey = 'AIzaSyAwKdrqS2GfCt9b2K1wAopDc9Ga0N1BVUM';
-const googleMapURL = `https://maps.googleapis.com/maps/api/js?key=${googleMapKey}&v=3.exp&libraries=geometry,drawing,places`;
-
-const Map = withScriptjs(withGoogleMap(props => (
-  <GoogleMap
-    {...props}
-    ref={props.mapRef}
-  >
-    <MarkerClusterer
-      averageCenter={true}
-      enableRetinaIcons={true}
-      gridSize={60}
-      ref={props.onMarkerClick}
-      defaultMaxZoom={16}
-    >
-      () = > {
-        this.orgMarkers = []
-      }
-
-      () => {
-        Object.entries(props.locationAddressHashTable).forEach( ([hashIndex,  orgRefs]) => {
-
-          this.orgMarkers.push(<OrganizationMarker
+const Map = withScriptjs(withGoogleMap( props => (
+        <GoogleMap
+            {...props}
+            ref={props.mapRef}
+        >
+            <MarkerClusterer
+                averageCenter={true}
+                enableRetinaIcons={true}
+                gridSize={60}
+                ref={props.onMarkerClick}
+                defaultMaxZoom={16}
+            >
+                () = > {
```

Comment:
It isn't clear to me what this is supposed to be doing here -- is it initializing something on the GoogleMap component ?